applications/nrf5340_audio/src/main.c

```diff
@@ -187,6 +189,9 @@ int main(void)
 
 		ret = zbus_chan_add_obs(&bt_mgmt_chan, &bt_mgmt_evt_sub, K_MSEC(200));
 		ERR_CHK(ret);
+
+		ret = zbus_chan_add_obs(&volume_chan, &volume_evt_sub, K_MSEC(200));
+		ERR_CHK(ret);
```

Comment:
Side question:  curious, since we involved ZBUS, we have to create multiple threads for observation, would that make the  CPU more busy, consume more stack or any drawback could happen? Or the affect is slightly.

Reply:
I guess there will be slightly more overhead when scheduling in and out of threads, but it shouldn't be a  massive change. Most of the threads created related to this is also low in priority, so the main important threads like encoding etc will take over when they need to